src/FrontDoor/FrontDoor/Models/PSRoutingRule.cs

```diff
@@ -17,34 +17,59 @@
 using System.Text.RegularExpressions;
 
 namespace Microsoft.Azure.Commands.FrontDoor.Models
-{    
-    /// <summary>
-        /// Represents the properties of an Azure Front Door object.
-    /// </summary>
-    public class PSRoutingRule: PSResource
+{
+    public class PSRouteConfiguration
     {
-        public List<string> FrontendEndpointIds { get; set; }
 
-        public List<PSProtocol> AcceptedProtocols { get; set; }
```

Comment:
You will need to add breaking change suppressions for removed properties.  See the BreakingChaingeIssues.xlsx in the failed ci checks